README-ja.md

```diff
@@ -1434,36 +1434,36 @@ POST /anotheroperation
 }
 ```
 
-RPCは振る舞いを公開することに焦点を当てています。RPCは内部通信パフォーマンスを理由として使われることが多いです。というのも、使用する状況に合わせてネイティブコールを自作することができるからです。
+RPCは振る舞いを公開することに焦点を当てています。RPCはそのパフォーマンスを理由に、内部通信においてよく利用されます。場合によっては、ユースケースに合わせてネイティブコールを自作することもできます。
 
-ネイティブライブラリー (aka SDK) を呼ぶのは以下の時:
+次のような場合はネイティブライブラリー (SDKとも呼ぶ) を使うほうがいいでしょう:
 
-* ターゲットのプラットフォームを知っている時
-* ロジックがどのようにアクセスされるのかを管理したいとき
-* ライブラリー外でエラーがどのようにコントロールされるかを管理したい時
-* パフォーマンスとエンドユーザーエクスペリエンスが最優先の時
+* ターゲットのプラットフォームが分かっているとき
+* 「ロジックに」対するアクセス方法を制御したいとき
+* ライブラリーの外側でエラーがどのようにコントロールされるかをを制御したいとき
+* パフォーマンスとエンドユーザーエクスペリエンスが最優先のとき
 
-**REST** プロトコルに従うHTTP APIはパブリックAPIにおいてよく用いられます。
+公開APIにおいては、**REST** に基づくHTTP APIの方がよく用いられています。
 
 #### 欠点: RPC
 
-* RPCクライアントとはサービス実装により厳密に左右されることになります。
-* 新しいオペレーション、使用例があるたびに新しくAPIが定義されなければなりません。
+* RPCクライアントがサービスの実装に密結合してしまいます。
+* 新しい操作やユースケースができるたびに、新しいAPIを定義しなければなりません。
 * RPCをデバッグするのは難しい可能性があります。
-* 既存のテクノロジーをそのまま使ってサービスを構築することはできないかもしれません。例えば、[Squid](http://www.squid-cache.org/)などのサーバーに[RPCコールが正しくキャッシュ](http://etherealbits.com/2012/12/debunking-the-myths-of-rpc-rest/) されるように追加で骨を折る必要があるかもしれません。
+* 既存のテクノロジーをそのまま活用することはできないかもしれません。例えば、[Squid](http://www.squid-cache.org/) などのサーバーに[RPCコールを正しくキャッシュさせる](http://etherealbits.com/2012/12/debunking-the-myths-of-rpc-rest/) には追加の作業が必要になるかもしれません。
 
 ### Representational state transfer (REST)
 
-RESTは、クライアントがサーバーによってマネージされるリソースに対して処理を行うクライアント・サーバーモデルを支持するアーキテキチャスタイルです。サーバーは操作できるもしくは新しいリソースレプレゼンテーションを受け取ることができるようなリソースやアクションのレプレゼンテーションを提供します。すべての通信はステートレスでキャッシュ可能でなければなりません。
+RESTはアーキテクチャスタイルの一つで、クライアント・サーバーモデルを推し進めたものです。クライアントは、サーバーが管理しているリソースに対して処理を行います。サーバーはリソースやアクションに対する「表現」(representation) を提供し、クライアントはそれに対する操作を行ったり、新しいリソースの表現を受け取ったりできます。すべての通信はステートレスでキャッシュ可能でなければなりません。
 
 RESTful なインターフェースには次の四つの特徴があります:
 
-* **特徴的なリソース (URI in HTTP)** - どのオペレーションであっても同じURIを使う。
-* **HTTP動詞によって変わる (Verbs in HTTP)** - 動詞、ヘッダー、ボディを使う
-* **自己説明的なエラーメッセージ (status response in HTTP)** - ステータスコードを使い、新しく作ったりしないこと。
-* **[HATEOAS](http://restcookbook.com/Basics/hateoas/) (HTML interface for HTTP)** - 自分のwebサービスがブラウザで完全にアクセスできること。
+* **リソース(HTTPにおけるURI)によって識別される** - どのオペレーションであっても同じURIを使うこと。
+* **表現(HTTPにおける動詞)によって変化する** - 動詞、ヘッダー、ボディを使うこと。
```

Comment:
'HTTP verb' vs 'HTTP method' again. I leave this argument for the English version.